src/NUnitFramework/framework/Attributes/TestOfAttribute.cs

```diff
@@ -33,7 +33,7 @@
 namespace NUnit.Framework
 {
     /// <summary>
-    /// Indicates which class the test or test fixture is testing
+    /// Indicates the method or class the test or test fixture is testing
```

Comment:
Assembly? We should also end the sentence with a comma.

Reply:
@mikkelbu You meant, end the sentence with a period?

Reply:
Yes :)